### ISSUE TITLE
CONSIDER wrapping console output to 80 characters

### ISSUE THREAD

**Issue by egnor** [robgonnella/ardi#51]
(Thanks for ardi! It is exactly what I was planning to write, but already written! 🙏)

This is minor and please feel free to ignore (not meaning to start a holy war), but on my regular xterms, ardi's output looks like this:

```
% ardi attach-and-watch --help

Compile, upload, watch board logs, and watch for sketch changes. Updates to the 
sketch file will trigger automatic recompile, reupload, and restarts the board l
og watcher. If the sketch argument matches a user defined build in ardi.json, th
e build values will be used for compilation, upload, and watch path

...
```

... which is a bit unattractive and hard to read, compared to word-wrapped text. Maybe you run everything in some terminal environment that always word-wraps?

**Comment by robgonnella** [comment]
Thanks for your suggestion, and thanks for trying out ardi! You're right this would be a good thing to get right. I'll add it to my list :)

**Comment by robgonnella** [comment]
Updated in new beta version, you can try it out via `GO111MODULE=on go get github.com/robgonnella/ardi/v2@v2.3.0-beta.1`

**Comment by egnor** [comment]
Looks good!! :+1:

**Comment by robgonnella** [comment]
Closing: Addressed in v2.3.0
